sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -495,6 +509,9 @@ def fit(self, X, y, sample_weight=None):
             print("Fitting gradient boosted rounds:")
 
         n_samples = X_binned_train.shape[0]
+        compute_raw_predictions_val = self._use_validation_data and isinstance(
```

Comment:
Without make the name `compute_raw_predictions_val` longer, can we rename it a bit à la "is_condition" to make clear it's a bool, e.g. `need_raw_predictions_val`?